third_party/move/documentation/coding_guidelines.md

```diff
@@ -23,6 +23,29 @@ $ cargo xclippy --all-targets
 
 In general, we follow the recommendations from [rust-lang-nursery](https://rust-lang.github.io/api-guidelines/) and [The Rust Programming Language](https://doc.rust-lang.org/book/).  The remainder of this guide provides detailed guidelines on specific topics in order to achieve uniformity of the codebase.
 
+## Code review
+
+We require every PR to have approval by at least two reviewers. This is enforced by the CI presubmit.
+
+### Guidelines for Authors of PRs
+
+- Have a meaningful description of the PR
+    - the purpose of a PR
+    - major design decisions
+    - testing strategy
+    - any additional, non-primary changes related to the PR. It is OK to do minor refactorings of the code as you visit it.
+- Respond to each comment of the reviewers. If you think you resolved a request for a change, just use 'done' as a response. If you diasgree with the reviewer, explain why
```

Comment:
diasgree--> disagree

Reply:
nit: missing . in the end of the sentence?

Reply:
Done